spec/controllers/bulk_upload_lettings_results_controller_spec.rb

```diff
@@ -5,6 +5,19 @@
     sign_in user
   end
 
+  describe "#show" do
+    let(:user) { create(:user) }
+    let(:bulk_upload) { create(:bulk_upload, :lettings, user:) }
+
+    it "passes thru pundit" do
+      allow(controller).to receive(:authorize)
+
+      get :show, params: { id: bulk_upload.id }
+
+      expect(controller).to have_received(:authorize)
+    end
+  end
+
   describe "GET #resume /lettings-logs/bulk-upload-results/:ID/resume" do
```

Comment:
Do we want to add a test here for #summary as well?

Reply:
this should be covered by the additions to `spec/requests/bulk_upload_lettings_results_controller_spec.rb` even though its a bit asymmetrical